CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@
     commit message, wrapped at 72 characters. Do not commit changes to
     __dist/ramda.js__.
 
-5.  Run `make test lint`, (or `grunt test`) and address any errors. Preferably,
+5.  Run unit tests by `make test lint` and address any errors. Preferably,
```

Comment:
This doesn't read well to me. I'd either remove the _unit tests by_ or begin with _Run unit tests by running_.